### ISSUE TITLE
Godot 3.4 beta 4 - room and portal postfixes not importing correctly

### ISSUE THREAD

**Issue by NineAlexT** [godotengine/godot#52322]
### Godot version

3.4 beta 4

### System information

Kubuntu 20.04

### Issue description

When importing a .glb from blender that uses the postfixes -room, -portal, and -bound, the objects are not imported correctly and remain  Spatials with -room, and MeshInstances with -portal and - bound at the end of their name instead of being converted to the correct type of nodes.

### Steps to reproduce

Create a Blender file with 2 empties, RoomA-room, and RoomB-room. parent some room geometry to them, some geometry for the bounds named RoomABounds-bound, and RoomBBounds-bound, and some geometry to one named RoomARoomBPortal-portal. Export the scene as a glb, and import it into Godot 3.4 beta 4.
![image](https://user-images.githubusercontent.com/60285548/131724035-ddad9714-9113-4e8f-a627-b178f3d0c71c.png)

### Minimal reproduction project

[3.4 beta test.zip](https://github.com/godotengine/godot/files/7093441/3.4.beta.test.zip)

**Comment by lawnjelly** [comment]
It actually seems to work fine (except the portal link isn't working, but you can flip that portal, as it is not pointing out from the source room).

The confusion is something I need to make more clear, the conversion of the node types doesn't happen during Godot import, it only happens the first time you run `convert rooms`. So if you create a scene, add the glb and add a RoomManager, set the roomlist then press rooms_convert, it converts successfully. :+1:

**Comment by NineAlexT** [comment]
Ah, that makes sense now! yea, if possible it'd be nice if this could work like the other postfix import conversions like -col, -colonly etc, but if not the documentation could at least be more clear since at the moment it says "The first time Godot encounters these specially named nodes, it will convert them..." So given the rest of the documentation I don't think it was too unreasonable of me to assume the conversion happens on import.

**Comment by lawnjelly** [comment]
Yeah it was well worth bringing this up, I'll check the other functionality today, and update the docs to make this more clear that changing the node type only happens during the room conversion.

I'll have a look at the import options, but it may not be easy to do at the import stage, because changing the node types relies on information that is calculated during the whole conversion process. For instance portals decide which rooms to link to by looking up the names on an previously calculated list of rooms, etc, etc. In short the nodes may not be able to be converted independently of the other nodes in the system, which makes things considerably more complex than e.g. a collision mesh or something.
